File: codes/java/src/main/java/com/github/nicolas_grevin/green_it/model/ProbeServiceStatus.java
package com.github.nicolas_grevin.green_it.model;

final public class ProbeServiceStatus {
  private String status;

  private String message;

  public ProbeServiceStatus() {

  }

  public ProbeServiceStatus(String status, String message) {
    super();
    this.status = status;
    this.message = message;
  }

  public String getStatus() {
    return this.status;
  }

  public ProbeServiceStatus setStatus(String status) {
    this.status = status;

    return this;
  }

  public String getMessage() {
    return this.message;
  }

  public ProbeServiceStatus setMessage(String message) {
    this.message = message;

    return this;
  }
}

File: codes/php/src/Entity/ProbeServiceStatus.php
<?php

declare(strict_types=1);

namespace App\Entity;

final class ProbeServiceStatus
{
    private string $status = "ok";
    private string $message = "Service is up";

    public function getStatus(): string
    {
        return $this->status;
    }

    public function getMessage(): string
    {
        return $this->message;
    }

    public function setStatus(string $status): ProbeServiceStatus
    {
        $this->status = $status;

        return $this;
    }

    public function setMessage(string $message): ProbeServiceStatus
    {
        $this->message = $message;

        return $this;
    }
}
